Community/News/20210407_GitHub_Analysis_Report.md

```diff
@@ -0,0 +1,36 @@
+## **Check Out The GitHub 2020 Digital Insight Report!**
+
+By Xiaoya
+
+This is a report produced by [X-lab](http://www.x-lab.info/) and jointly completed with several scientific institutions. We wish to share the insights and metrics that were digged out through GitHub collaboration data with CHAOSS.
+
+Open source software has become a cornerstone of our digital world, and open collaboration plays an enormous role in the development of human digital civilization.
+
+**GitHub**, the world's largest open source collaboration platform, produced a massive amount of developer behavior data.
+
+We obtained **860 million** event logs generated in 2020 and analyzed **14.54 million** active developers and **54.21 million** active projects.
+
+What have we learned?
+
+Please read our [GitHub 2020 Digital Insight Report](http://oss.x-lab.info/github-insight-report-2020-en.pdf).
+
+![2021040601](/Users/xiaoya/Documents/GitHub/website/Community/News/images/2021040601.jpg)
```

Comment:
Please make path relative to the GitHub repo

Reply:
Thanks Georg. I've made those changes.

Reply:
Awesome, thanks!